### DISCUSSION TITLE
Style Height

### DISCUSSION THREAD

**Discussion by jsbowling42** [piitaya/lovelace-mushroom#789] [Q&A] [answered]
is there a way to adjust the light cards height? when i add the code below it doesn't change the height. it seems to still be using the "constructed stylesheet" height property from what i would presume is inherited from the parent object. I've attached an image to show the area I want to fill.

 style: |
      ha-card {
        height: 100%;

![Capture](https://user-images.githubusercontent.com/35424582/197879313-7e33740e-db5c-4c5f-8544-afd1a9e4583d.PNG)

**Comment by gfandrea** [comment, marked as answer]
Are those light card inside a grid card?
![image](https://user-images.githubusercontent.com/34663993/198028681-d1c20320-9381-464f-8973-62711cf5d7b2.png)

have you tried with the "fill the container" toggle?
![image](https://user-images.githubusercontent.com/34663993/198028834-8f70e980-cec3-4500-91df-1c84cf00e068.png)

**Reply by jsbowling42** [reply]
Yes, the light cards are in a grid card. I have tried that toggle. though its called "Render cards as squares", which I assume is just a translation issue. when i set the toggle, all the cards increase and there is excess space on both now. see screenshot. 
![Capture2](https://user-images.githubusercontent.com/35424582/198032415-99c49b2a-f5bd-4c35-9326-e9fc6cfe5cdb.PNG)

**Comment by gfandrea** [comment]
No, the one I am talking about is an option of the mushroom light card next to where you choose the disposition (vertical in your case)
In my example the "Render cards as squares" toggle is off

**Reply by jsbowling42** [reply]
ah, I never saw that toggle before (built in Yaml). i should have looked in the light card variables page. that fixed it. thanks.
